firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/ShakeForFeedback.kt

```diff
@@ -9,9 +9,46 @@ import com.google.firebase.appdistribution.ktx.appDistribution
 import com.google.firebase.ktx.Firebase
 import com.squareup.seismic.ShakeDetector
 
-class ShakeForFeedback private constructor() : ShakeDetector.Listener,
-    Application.ActivityLifecycleCallbacks {
+object ShakeForFeedback : ShakeDetector.Listener, Application.ActivityLifecycleCallbacks {
+    const val TAG: String = "ShakeForFeedback"
+
     private val shakeDetector = ShakeDetector(this)
+    private var isEnabled = false
+
+    fun enable(application: Application) = enable(application, null)
```

Comment:
I think you don't need this overload by using [default arguments](https://kotlinlang.org/docs/functions.html#default-arguments) - i.e. `null` for `currentActivity` in the function below

Reply:
Kotlin FTW! Thanks